modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -662,12 +675,34 @@ function getIssueFields()
         $issueID   = $_GET['issueID'];
         $issueData = getIssueData($issueID);
 
-        $desc       = $db->pselect(
+        $desc = $db->pselect(
             "SELECT issueComment
 FROM issues_comments WHERE issueID=:i
 ORDER BY dateAdded LIMIT 1",
             array('i' => $issueID)
         );
+
+        $attachments = $db->pselect(
+            "SELECT
+                        ID,
```

Comment:
You can use SELECT * if you're going to include all the columns.

Reply:
@johnsaigle it was suggested from @driusan to not use *.